workspaces/ui-v2/src/entry-points/cloud/cloud-viewer.tsx

```diff
@@ -73,11 +73,14 @@ export default function CloudViewer() {
           return process.env.REACT_APP_STAGING_API_BASE;
         }
       })();
-      const response = await fetch(`${apiBase}/api/specs/${specId}`, {
-        headers: { accept: 'application/json' },
-      });
+      const response = await fetch(
+        `${apiBase}/api/specs/${personId}/${specId}`,
```

Comment:
similar comment here; this resource hierarchy is inconsistent and doesn't accommodate orgs